### PR TITLE
Update run-slam-cartographer.md

### PR DESCRIPTION
**Pull request by tessavitabile**
Documenting a known issue with rerunning offline mode

### CODE REVIEW

File: docs/services/slam/run-slam-cartographer.md [hunk 1]
```diff
@@ -338,6 +338,12 @@ Only `2D SLAM` is currently implemented for Cartographer.
 Because of this, Cartographer assumes your Rplidar will remain at roughly the same height while in use.
 If maps are not building the way you expect, make sure your Rplidar is secure and at roughly the same height throughout the run.
 
+### Issue: Offline mode produces an error after restart
+
+If there is a saved map in `data_dir/map` and saved data in `data_dir/data` from a previous run, then offline mode
+may error at startup, since the data has already been incorporated into the map. If that occurs, you can
+either clear `data_dir/map` to rerun offline mode, or clear `data_dir/data` to view the saved map.
```
Comment (JohnN193) [inline docs/services/slam/run-slam-cartographer.md:345]: Clearing the data directory does not work for this, as cartographer will throw the error `terminating with uncaught exception of type std::runtime_error: not enough data in data directory`
```suggestion
clear `data_dir/map` to rerun the dataset in offline mode.
```

**Review by JohnN193 — CHANGES_REQUESTED** [review]
Unfortunately only clearing the map directory is a possible fix for this issue currently

File: docs/services/slam/run-slam-cartographer.md (hunk 1 above)
Comment (tessavitabile) [reply docs/services/slam/run-slam-cartographer.md:345]: ah, thank you

**Review by npentrel — APPROVED** [review]
Thank you!

**Pull request merged**